gremlint/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gremlint",
-  "version": "1.0.0",
+  "version": "3.5.1-alpha1",
```

Comment:
it doesn't appear as though we've ever released an actual "alpha" under gremlin-javascript. we have done an "rc" though in which case it has gone out as "rc1":

https://www.npmjs.com/package/gremlin/v/3.4.0-rc1

so i assume alpha1 would work ok. technically, i think we'd try to publish as an "rc1" rather than "alpha1" but i suppose that is besides your point. we can change that as necessary when we discuss the early test release on the dev list.